packages/runtime/src/microapp.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { MicroappRuntime } from './microappRuntime';
+
+type MicroappProps = {
+  src: string;
```

Comment:
Consistency with the `Runtime` class.

```suggestion
  url: string;
```